src/pal/src/thread/process.cpp

```diff
@@ -2861,6 +2869,60 @@ void PROCNotifyProcessShutdown()
 }
 
 /*++
+Function
+  PROCAbortInitialize()
+  
+Abstract
+  Initialize the process abort crash dump program file path and
+  name. Doing all of this ahead of time so nothing is allocated
+  or copied in PROCAbort/signal handler.
+  
+Return
+  TRUE - succeeds, FALSE - fails
+  
+--*/
+BOOL
+PROCAbortInitialize()
+{
+    if (g_szCoreCLRPath == nullptr)
+    {
+        return FALSE;
+    }
+    char program[MAX_LONGPATH];
```

Comment:
I would prefer not to use fixed length limits on paths. We have tried to erradicate all such limits in the past. We could allocate this buffer dynamically here since we know the size we need. Then we can pass the dynamically allocated  buffer to the g_argvCreateDump[0].